### DISCUSSION TITLE
After build automatic deleted old uploaded files.

### DISCUSSION THREAD

**Discussion by mdjamilaj** [adonisjs/core#4015] [Help]
Upload file code.

`          
      await file.move(Application.tmpPath('uploads'), {
            name:Math.floor(Math.random() * 123456789) + new Date().getTime() + "." + file.extname
      })
`

It's working well normally. When I update something and rebuild. Old all files didn't find my old file.

I check why the file was uploaded under the build folder.

Anyone helps me to fix it?

**Comment by Barbapapazes** [comment]
Hello,

I think that I understand but can you give a list of instruction (things you do) which leads to deleted files?


(what did you mean by update something for exemple?)
